operations.py

```diff
@@ -0,0 +1,293 @@
+        return self._sensor_isolation_by_ip(params,'/rest/monitor/global/commands/isolate')
+
+
+    def unisolate_sensor_by_ip(self,params):
+        '''
+        Un-isolate a sensor using its internal IP address
+        '''
+        return self._sensor_isolation_by_ip(params,'/rest/monitor/global/commands/un-isolate')
+
+    def kill_process(self,params):
+        '''
+        Kill a process identified by process_id on machine_id
+        '''
+        machine_guid = str(params.get('machine_guid'))
+        process_guid = str(params.get('process_guid'))
+        json_body = {
+            "initiatorUserName": self.username,
+            "actionsByMachine": {
+                machine_guid: [
+                    {
+                        "targetId": process_guid,
+                        "actionType": "KILL_PROCESS"
+                    }
+                ]
+            }
+        }
+        logger.debug('JSON Body: {}'.format(json_body))
+        return self.make_rest_call(endpoint='/rest/remediate', json=json_body)
+
+
+    def white_blacklist(self,params):
+        '''
+        Whitelist and Blacklist Hashes, IPs and Domains organization wide
+        '''
+        json_body = white_blacklist_body_builder(params)
+        logger.debug('JSON Body: {}'.format(json_body))
+        return self.make_rest_call(endpoint='/rest/classification/update', json=json_body)
+
+
+    def blacklist_file(self,params):
+        '''
+        wrapper for white_blacklist
+        '''
+        return self.white_blacklist(params)
+
+
+    def whitelist_file(self,params):
+        '''
+        wrapper for white_blacklist
+        '''
+        return self.white_blacklist(params)
+
+
+    def blacklist_ip_or_domain(self,params):
+        '''
+        wrapper for white_blacklist
+        '''
+        return self.white_blacklist(params)
+
+
+    def whitelist_ip_or_domain(self,params):
+        '''
+        wrapper for white_blacklist
+        '''
+        return self.white_blacklist(params)
+
+
+def _run_operation(config,params):
+    '''
+    Map operations to Cybereason methods
+    '''
+    operation = params['operation']
+    cr_object = CybereasonMC(config)
+    command = getattr(CybereasonMC,operation)
+    response = command(cr_object,params)
+    cr_object.close()
+    return response
+
+
+def _check_health(config):
+    '''
+    Test service availability with a login/logoff
+    '''
+    try:
+        cr_object = CybereasonMC(config)
+        server_config = cr_object.make_rest_call(endpoint='/rest/settings/configurations', method='GET')
+        cr_object.close()
+        if server_config['status'] == 'Failure':
+            logger.exception('Authentication Error, Check URL and Credentials')
+            raise ConnectorError('Authentication Error, Check URL and Credentials')
+
+    except Exception as Err:
+        logger.exception('Health Check Error:{}'.format(Err))
+        raise ConnectorError('Health Check Error:{}'.format(Err))    
+
+operations = {
```

Comment:
we don't need this map. could directly call _run_operation(params) from connector.py